src/azure-cli/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -212,7 +212,7 @@ def _mysql_auto_grow_validator(auto_grow, replication_role, high_availability, i
         replication_role = instance.replication_role if replication_role is None else replication_role
         high_availability = instance.high_availability.mode if high_availability is None else high_availability
     # if replica, cannot be disabled
-    if replication_role != 'None' and auto_grow.lower() == 'disabled':
+    if replication_role is not None and auto_grow.lower() == 'disabled':
```

Comment:
```suggestion
    if replication_role is not None and replication_role != 'None' and auto_grow.lower() == 'disabled':
```
Not sure if it could be `None` string. Better to keep the original logic

Reply:
@DaeunYim for review

Reply:
![image](https://user-images.githubusercontent.com/69321306/176571707-5f9119a5-3112-4ed9-934c-7a7ad3ba1a87.png)
![image](https://user-images.githubusercontent.com/69321306/176571733-b61436d5-c7b2-40b6-ab4c-95bc000ab7b1.png)
It is translated as "None" string value